static/js/code-highlighter.js

```diff
@@ -496,3 +496,17 @@ $(function () {
   });
 
 });
+
+// We use user-select:none to hide line numbers from text selections. That
+// does however not work in Chrome (https://crbug.com/850685)
+// or Safari (https://bugzilla.mozilla.org/show_bug.cgi?id=1616104#c1).
+// As a work-around, move all line numbers into pseudo-elements when the user
+// selects something for the first time.
+if (navigator.userAgent.indexOf('Firefox') == -1) {
+  document.addEventListener('selectstart', function() {
```

Comment:
Can we just move the line-number to be a pseudo-element instead? Maybe a CSS counter?

Reply:
The line number is already moved to a pseudo-element.

Does a CSS counter have any advantage over the attribute? Improved performance, possibly?

The change would be trivial, so let me know what you prefer. The current PR or just emptying the textContent, combined with the following style:

```
#file {
  counter-reset: lineno;
}
.line-number:empty::before {
  content: counter(lineno);
  counter-increment: lineno;
}
```

Reply:
I mean, can we always use a pseudo-element, for all browsers, rather than using the UA string?

Reply:
We could. It's basically a revert of https://bugzilla.mozilla.org/show_bug.cgi?id=1558942

I didn't measure the impact on performance or other side effects.

The only "feature" that I know of is that the current implementation allows Ctrl-F to find line numbers, whereas browser search won't find content in pseudo-elements.